inappmessaging/USERGUIDE.md

```diff
@@ -218,6 +218,19 @@ InAppMessaging.instance().onVerifyContext = { contexts: List<String>, campaignTi
 }
 ```
 
+### <a name="close-campaign"></a> #2 Close campaign API
+There are cases that apps need to manually close the campaigns without user interaction.
```

Comment:
There are cases that apps -> There may be cases where apps

Reply:
Changed to the suggested phrase.